internal/rbac/rbac.go

```diff
@@ -0,0 +1,84 @@
+package rbac
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	kmmv1beta1 "github.com/kubernetes-sigs/kernel-module-management/api/v1beta1"
+)
+
+//go:generate mockgen -source=rbac.go -package=rbac -destination=mock_rbac.go
+
+type RBACCreator interface {
+	CreateModuleLoaderRBAC(ctx context.Context, mod kmmv1beta1.Module) error
+	CreateDevicePluginRBAC(ctx context.Context, mod kmmv1beta1.Module) error
```

Comment:
What do you think about 
```suggestion
	CreateModuleLoaderServiceAccount(ctx context.Context, mod kmmv1beta1.Module) error
	CreateDevicePluginServiceAccount(ctx context.Context, mod kmmv1beta1.Module) error
```
?

We are basically creating a ServiceAccount here.

Reply:
Makes total sense, I updated the PR accordingly :+1: